static/bidder-info/rtbhouse.yaml

```diff
@@ -1,4 +1,16 @@
+# Contact prebid@rtbhouse.com to ask about enabling a connection to the bidder.
+# We have the following regional endpoint urls
+# Please uncomment the appropriate regional endpoint for your datacenter
```

Comment:
Suggesting to rename this commnet to - `Please configure the following endpoints for your datacenter`